usermanager/src/test/java/org/pesmypetcare/usermanager/clients/pet/PetCollectionsManagerClientTest.java

```diff
@@ -0,0 +1,644 @@
+                + "      \"description\": \"I'm not feeling so well\",\n"
+                + "      \"type\": \"Normal\",\n"
+                + "      \"severity\": \"High\"\n"
+                + "    },\n"
+                + "    \"key\": \"1995-01-08T15:20:30\"\n"
+                + "  },{\n"
+                + "    \"body\": {\n"
+                + "      \"endDateTime\": \"2000-01-08T15:20:30\",\n"
+                + "      \"description\": \"I'm not feeling so well\",\n"
+                + "      \"type\": \"Normal\",\n"
+                + "      \"severity\": \"High\"\n"
+                + "    },\n"
+                + "    \"key\": \"1998-01-08T15:20:30\"\n"
+                + "  }\n"
+                + "]");
+        illnessDataJson = new StringBuilder("{\n"
+                + "      \"endDateTime\": \"2000-01-08T15:20:30\",\n"
+                + "      \"description\": \"I'm not feeling so well\",\n"
+                + "      \"type\": \"Normal\",\n"
+                + "      \"severity\": \"High\"\n"
+                + "}"
+        );
+        illnessData = new IllnessData("2000-01-08T15:20:30", "I'm not feeling so well",
+                IllnessType.Normal, SeverityType.High);
+        illnessCollectionList = new ArrayList<>();
+        illnessCollectionList.add(new Illness(DATE_1, illnessData));
+        illnessCollectionList.add(new Illness(DATE_2, illnessData));
+        illnessCollectionList.add(new Illness(DATE_3, illnessData));
+
+        mealCollectionJson = new StringBuilder("[\n"
+                + "  {\n"
+                + "    \"body\": {\n"
+                + "      \"kcal\": 85.44,\n"
+                + "      \"mealName\": \"Tortilla\"\n"
+                + "    },\n"
+                + "    \"key\": \"1990-01-08T15:20:30\"\n"
+                + "  },{\n"
+                + "    \"body\": {\n"
+                + "      \"kcal\": 85.44,\n"
+                + "      \"mealName\": \"Tortilla\"\n"
+                + "    },\n"
+                + "    \"key\": \"1995-01-08T15:20:30\"\n"
+                + "  },{\n"
+                + "    \"body\": {\n"
+                + "      \"kcal\": 85.44,\n"
+                + "      \"mealName\": \"Tortilla\"\n"
+                + "    },\n"
+                + "    \"key\": \"1998-01-08T15:20:30\"\n"
+                + "  }\n"
+                + "]");
+        mealDataJson = new StringBuilder("{\n"
+                + "  \"kcal\": 85.44,\n"
+                + "  \"mealName\": \"Tortilla\"\n"
+                + "}"
+        );
+        mealData = new MealData("Tortilla", 85.44);
+        mealCollectionList = new ArrayList<>();
+        mealCollectionList.add(new Meal(DATE_1, mealData));
+        mealCollectionList.add(new Meal(DATE_2, mealData));
+        mealCollectionList.add(new Meal(DATE_3, mealData));
+
+        medicationCollectionJson = new StringBuilder("[\n"
+                + "  {\n"
+                + "    \"body\": {\n"
+                + "      \"quantity\": 85.0,\n"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "      \"quantity\": 85.0,\n" appears 4 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5549434)